packages/superset-ui-legacy-plugin-chart-parallel-coordinates/src/vendor/parcoords/d3.parcoords.css

```diff
@@ -41,22 +43,26 @@
   opacity: 0.25;
 }
 .parcoords {
-	-webkit-touch-callout: none;
-	-webkit-user-select: none;
-	-khtml-user-select: none;
-	-moz-user-select: none;
-	-ms-user-select: none;
-	user-select: none;
-    background-color: white;
+  -webkit-touch-callout: none;
+  -webkit-user-select: none;
+  -khtml-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
```

Comment:
Just realizing we're not using autoprefixer... I'll try to make a PR around that soon :D